platformproxy/platformproxy.pro

```diff
@@ -24,7 +24,12 @@ BUILD_ROOT = $$clean_path( $$OUT_PWD/../src )
 INCLUDEPATH *= $${PROJECT_ROOT}
 DEPENDPATH *= $${PROJECT_ROOT}
 
-LIBS *= -L$${BUILD_ROOT}/lib -lvncgl
+CONFIG(debug, debug|release) {
+    LIBS *= -L$${BUILD_ROOT}/lib -lvncgld
+}
+else {
+    LIBS *= -L$${BUILD_ROOT}/lib -lvncgl
+}
```

Comment:
Using qtLibraryTarget from qt_functions.pri should cover even more platforms. Guess we have to use qt5LibraryTarget for Qt5 builds.

BTW: in case you have some time for this project: https://github.com/uwerat/vnc-eglfs/issues/12 ?

Reply:
I haven't worked with qmake for a couple of years... you're right, using qt5LibraryTarget is way simpler. That should work for Qt6 also. 
Regarding CMake support I can't make any promises, since my time is limited currently.

Reply:
using qtLibraryTarget now